template/faas-flow/main.go

```diff
@@ -1,9 +1,16 @@
 package main
 
 import (
-	"handler/controller"
+	"handler/config"
+	"handler/openfaas"
+	"handler/runtime/controller"
+	"log"
 )
 
 func main() {
-	controller.StartServer()
+	runtime := &openfaas.OpenFaasRuntime{}
+	port := 8082
+	readTimeout := config.ReadTimeout()
```

Comment:
Just out of curiosity, is there any specific reason to move out the config as well?

Reply:
I think given the runtime user might like to configure It accordingly. For example in openfaas timeout needs to be aligned with timeout configured in of-watchdog highest being 120sec. A platform that allows more that than might allow longer timeout